diff --git a/node_modules/@types/globalize/dist/globalize/currency.d.ts b/node_modules/@types/globalize/dist/globalize/currency.d.ts
index 4656464..9560501 100644
--- a/node_modules/@types/globalize/dist/globalize/currency.d.ts
+++ b/node_modules/@types/globalize/dist/globalize/currency.d.ts
@@ -6,6 +6,10 @@ declare module "../globalize" {
 		* symbol (default), accounting, code or name.
 		*/
 		style?: "symbol" | "accounting" | "code" | "name";
+		/**
+		 * enable alt-narrow currency symbol
+		 */
+		symbolForm?: 'narrow';
 	}
 
 	interface Shared {
diff --git a/node_modules/@types/globalize/dist/globalize/unit.d.ts b/node_modules/@types/globalize/dist/globalize/unit.d.ts
index 736f086..3359e21 100644
--- a/node_modules/@types/globalize/dist/globalize/unit.d.ts
+++ b/node_modules/@types/globalize/dist/globalize/unit.d.ts
@@ -12,7 +12,7 @@ declare module "../globalize" {
 		/**
 		 * numberFormatter: [Function] a number formatter function. Defaults to Globalize .numberFormatter() for the current locale using the default options.
 		 */
-		numberFormatter?: NumberFormatterOptions;
+		numberFormatter?: (value: number) => string;
 	}
 
 	interface Shared {
